src/llv8.cc

```diff
@@ -415,9 +433,13 @@ std::string JSFunction::Inspect(InspectOptions* options, Error& err) {
         res += source + "\n";
       }
     }
+    res = res + ">";
+  } else {
+    std::stringstream ss;
+    ss << rang::fg::yellow << res << ">" << rang::fg::reset;
+    res = ss.str().c_str();
```

Comment:
At line 396 you called `res = ss.str();` (without `.c_str()`). I guess this should work here as well.